packages/react-tabs/.eslintrc.json

```
@@ -0,0 +1,8 @@
+{
+  // OUFR has some overrides this package needs to mirror
+  "extends": ["../office-ui-fabric-react/.eslintrc"],
+  "root": true,
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "error"
+  }
```

Pretty sure you can just use the default config here:
```suggestion
  "extends": ["plugin:@fluentui/eslint-plugin/react"],
  "root": true
```

---

Done. I had to suppress some interface naming warnings that came up as a result.

---

I assume in a future change you're planning on actually renaming to Tabs? If so, that would also be a good time to fix the interface names (at least in the renamed versions).

---

Yup!